### DISCUSSION TITLE
Support for multi-modal models in Playground?

### DISCUSSION THREAD

**Discussion by leostudiooo** [community/community#138705] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

It gets me thinking when I head to multi-modal models such as OpenAI's GPT-4o or Phi-3.5 vision instruct, where there aren't any "add attachment" icons or buttons in the Playground input box. 
<img width="1436" alt="image" src="https://github.com/user-attachments/assets/47ca4d50-5b82-4072-a364-e5f9b2ee2db3">

Is this going to be a feature supported in future versions of Models Playground? Or if it's that I just haven't discovered the option yet?

**Comment by matthewisabel** [comment, marked as answer]
Multi-modal input is coming soon!
